src/_colors.scss

```diff
@@ -0,0 +1,8 @@
+/* COLORS */
```

Comment:
is there any reason for the file to have `_` in the name?

Reply:
When I was learning about sass I found that imported file should have that underscore before filename so it would recognize it, but I guess it's old information, it's working without it so I renamed it

Reply:
It may have been just to separate scss files for css. It was totally fine like that as long as it was working. I was just surprised about reasoning